knotx-stack-manager/src/main/descriptor/knotx-stack.json

```diff
@@ -28,31 +28,37 @@
     },
     {
       "groupId": "io.knotx",
-      "artifactId": "knotx-knot-service",
+      "artifactId": "knotx-data-bridge-api",
       "version": "\${knotx.version}",
       "included": true
     },
     {
       "groupId": "io.knotx",
-      "artifactId": "knotx-knot-action",
+      "artifactId": "knotx-data-bridge-core",
       "version": "\${knotx.version}",
       "included": true
     },
     {
       "groupId": "io.knotx",
-      "artifactId": "knotx-knot-handlebars",
+      "artifactId": "knotx-data-bridge-http",
```

Comment:
i think naming the http adapter as `knotx-data-bridge-http` while we have `knotx-data-bridge-core` might be missleading. I think the better naming of http adapter should be `knotx-data-source-http`.
Additionally, it would be more clear to put any extension into each own groupId, e.g.
`groupId=io.knotx` - reserved for core
`groupid=io.knotx.databridge` - for data bridge and source adatpers
`groupid=io.knotx.xyz` - for other module that we will create in the future